### PR TITLE
Issue #71 Manage organization frontend

### PR DESCRIPTION
**Pull request by RobertBeaudenon**
User story 1.6
Provided the following functionalities:
- Created a new component that is accessible through the sidenav to hold organization management
- Inside the component you can:
 - Retrieve all organizations (allowed for SA and IM)
 - Create an organization (allowed for SA)
 - DELETE an organization (allowed for SA)
 - Update an organization (allowed for SA and IM)
- All errors are displayed (related to permissions and others)
- Tested service and component

(To update and delete you must click on the organization name first in the list)

### CODE REVIEW

**Review by NicholasNagy — CHANGES_REQUESTED** [review]
So, I think for the manage organizations user story, what we would like to have is on login as a System Admin, we would see the organizations. Then the SA would be able to choose which organization they would want to work with and then perform tasks like modifying users, looking at the dashboard, etc. exactly like an inventory manager.

File: client/src/app/components/login/login.component.ts [hunk 1]
```diff
@@ -40,10 +40,16 @@ export class LoginComponent implements OnInit {
...
+        this.tokenService.SetToken(data.token); // Setting token in cookie for logged in users   
+        if(data.role === 'SA') {
+          setTimeout(() => {
+            this.router.navigate(['manage-organizations']);
+          }, 1000);
+        }else {
         setTimeout(() => {
           this.router.navigate(['']); // Redirect user to component in path:home (defined in alta-home-routing.module.ts)
         }, 1000); // Redirect the user after 1 seconds ( in case we want to add a loading bar when we click on button )
+      }
```
Comment (NicholasNagy) [inline client/src/app/components/login/login.component.ts:52]: This line isn’t properly indented, or is it just me?

File: client/src/app/components/manage-organizations/manage-organizations.component.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          {{errorMessage}}
+</mat-error>
+
+<mat-divider></mat-divider>
+<mat-list  *ngFor="let organization of organizations">
+    <mat-list-item>
+        <h3  (click)="organizationClicked(organization)">{{ organization.org_name }}</h3>
+    </mat-list-item>
+    <mat-divider></mat-divider>
+    <!-- <li *ngFor="let organization of organizations">
```
Comment (NicholasNagy) [inline client/src/app/components/manage-organizations/manage-organizations.component.html:23]: Why is this code commented out?

File: client/src/app/components/manage-organizations/manage-organizations.component.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+         this.errorMessage = '';
+      },
+      (err) => {
+        this.errorMessage = err;
+      }
+    );
+  }
+
+  organizationClicked(organization): void {
+    // console.log(organization.org_id)
```
Comment (NicholasNagy) [inline client/src/app/components/manage-organizations/manage-organizations.component.ts:36]: Please remove this debugging statement.

**Review by NicholasNagy — COMMENTED** [review]
Generally, I think the logic of the code is sound. There are just a few areas where you forgot to delete the code you commented out.
Additionally, you should fix the linter warnings that the linter is flagging.

Once this is done, i will approve and merge, otherwise, this is a good pr, thanks!

File: client/src/app/components/login/login.component.ts (hunk 1 above)
Comment (RobertBeaudenon) [reply client/src/app/components/login/login.component.ts:52]: It's you.

File: client/src/app/components/manage-organizations/manage-organizations.component.html (hunk 2 above)
Comment (RobertBeaudenon) [reply client/src/app/components/manage-organizations/manage-organizations.component.html]: Fixed.

File: client/src/app/components/manage-organizations/manage-organizations.component.ts (hunk 3 above)
Comment (RobertBeaudenon) [reply client/src/app/components/manage-organizations/manage-organizations.component.ts]: Fixed.

**Comment by RobertBeaudenon** [comment]
Cleaned code based on PR review and lint. Should be good to merge.

**Review by NicholasNagy — APPROVED** [review]
Everything is looking good now, however, it will obviously need a UI overhaul when we've hashed out the mockups.

**Pull request merged**